### PR TITLE
Update ANU faculty.

### PR DESCRIPTION
**Pull request by hosking**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-g.csv [hunk 1]
```diff
@@ -622,6 +624,8 @@ Graham Kirby,University of St Andrews,https://graham.host.cs.st-andrews.ac.uk,_p
 Graham Knight,University College London,http://www.cs.ucl.ac.uk/staff/graham_knight,HdunwTkAAAAJ
 Graham Neubig,Carnegie Mellon University,http://www.phontron.com,wlosgkoAAAAJ
 Graham Roberts,University College London,http://www0.cs.ucl.ac.uk/staff/G.Roberts,NOSCHOLARPAGE
+Graham Williams,Australian National University,https://comp.anu.edu.au/people/graham-williams/,08HEV_kAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv]: Not listed as an alias in DBLP for Graham J. Williams; please remove.

File: csrankings-j.csv [hunk 2]
```diff
@@ -731,6 +731,7 @@ Jennifer Tour Chayes,Univ. of California - Berkeley,http://jenniferchayes.com,YA
 Jennifer Waycott,University of Melbourne,http://findanexpert.unimelb.edu.au/display/person52243,kc5Fr-0AAAAJ
 Jennifer Widom,Stanford University,http://infolab.stanford.edu/~widom,zdKmnYwAAAAJ
 Jenny Coady,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/jenny-coady.htm,NOSCHOLARPAGE
+Jennyfer Lawrence Taylor,Australian National University,https://comp.anu.edu.au/people/,YojqErAAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: Web page needs to point to specific faculty member.

File: csrankings-k.csv [hunk 3]
```diff
@@ -263,6 +263,7 @@ Katharina Krombholz,CISPA Helmholtz Center,https://cispa.saarland/people/kathari
 Katharina Morik,TU Dortmund,https://www-ai.cs.tu-dortmund.de/PERSONAL/morik.html,LfmjF2UAAAAJ
 Katharina Reinecke,University of Washington,https://www.cs.washington.edu/people/faculty/reinecke,_4EISRwAAAAJ
 Katharina Siobhan Kohls,Radboud University,https://kkohls.org/index.html,HQ8HblYAAAAJ
+Katherine A. Daniel,Australian National University,https://cybernetics.anu.edu.au/people/,TIQVsWEAAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv]: Web page needs to point to specific faculty member.

File: csrankings-m.csv [hunk 4]
```diff
@@ -1148,6 +1145,8 @@ Matthew Green 0001,Johns Hopkins University,https://isi.jhu.edu/~mgreen,X0XWAGkA
 Matthew Guzdial,University of Alberta,http://guzdial.com,jKqmTbIAAAAJ
 Matthew Hague,Royal Holloway University of London,https://pure.royalholloway.ac.uk/portal/en/persons/matthew-hague(c4baa4a0-192e-49c8-a699-feda6a04a583).html,s37BfYAAAAAJ
 Matthew Hicks,Virginia Tech,http://www.impedimenttoprogress.com,_rJCgzIAAAAJ
+Matthew Hole,Australian National University,https://comp.anu.edu.au/people/matthew-hole/,BoPimVUAAAAJ
```
Comment (emeryberger) [inline csrankings-m.csv:1152]: Not listed as an alias in DBLP. If they are the same person, please send the DBLP folks a note and copy me.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Hi @hosking - please address comments and I will merge - thanks!

File: csrankings-j.csv (hunk 2 above)
Comment (hosking) [reply csrankings-j.csv]: Done

File: csrankings-g.csv (hunk 1 above)
Comment (hosking) [reply csrankings-g.csv]: Done

File: csrankings-m.csv (hunk 4 above)
Comment (hosking) [reply csrankings-m.csv:1152]: Email attached
[Untitled.pdf](https://github.com/emeryberger/CSrankings/files/8401463/Untitled.pdf)
.

**Comment by hosking** [comment]
Changes made as requested.

**Pull request merged**